scripts/valhalla_build_timezones

```diff
@@ -5,16 +5,6 @@ error_exit() {
   exit 1
 }
 
-if [ -z "$1" ]; then
```

Comment:
ive changed the timezone building script to output the sqlite db to stdout so you can forward it to file like our other scripts. it was very annoying to have to have a config file just to pass it into the script. alternatively i could have updated it to have a file name for the output file. let me know if this is agreedous @gknisely

Reply:
looks like CI doesnt like this change...

Reply:
looks fine, but is it now working?  why do you cat the file at the end?

Reply:
the cat at the end is so that the "output" is on stdout essentially. why its not working, im not yet sure. it works locally so perhaps its something about CI not having the sqlite binary or wget or something installed. at the moment ive silenced the output so i cant see why it fails. ill unsilence it so i can see whats missing on CI